uw-frame-components/js/frame-config.js

```diff
@@ -1069,5 +1070,11 @@ define(['angular'], function(angular) {
             'description': 'This flag disables announcement group filtering ' +
               'if you have it enabled (page refresh required)',
           },
+          {
+            'id': 'optInAvatar',
+            'title': 'Allow Use Of My Photo',
+            'description': 'This flag allows use of your photograph ' +
+              'if you have it enabled (page refresh required)',
+          },
```

Comment:
Agree. Would be best for the user-facing opt-in / opt-out setting to be in the `/user-settings` page. Could hide the presentation of that setting behind its own feature flag, but probably better to just hide it behind whether the active theme supports avatar image feature.